apps/frontend/src/libs/modules/pwa/libs/enums/update-interval.enum.ts

```diff
@@ -0,0 +1,14 @@
+const TimeConstants = {
+	MILLISECONDS_IN_SECOND: 1000,
+	MINUTES_IN_HOUR: 60,
+	SECONDS_IN_MINUTE: 60,
+} as const;
+
+const UpdateInterval = {
+	DEFAULT:
+		TimeConstants.MINUTES_IN_HOUR *
+		TimeConstants.SECONDS_IN_MINUTE *
+		TimeConstants.MILLISECONDS_IN_SECOND,
+} as const;
```

Comment:
Time constant is not an enum by its logic, more like a set of mapped constants, so let's move it to frontend/libs/constants level. For the update interval, it should be in local constants and let's simplify it to singular value, I beleive we won't need to add other options
```suggestion
const TIME_UNIT = {
  MILLISECONDS_IN_SECOND: 1000,
  SECONDS_IN_MINUTE: 60,
  MINUTES_IN_HOUR: 60,
} as const;

const UPDATE_INTERVAL = TimeConstant.MINUTES_IN_HOUR *
    TimeConstant.SECONDS_IN_MINUTE *
    TimeConstant.MILLISECONDS_IN_SECOND;
```